assets/airports/eidw.json

```diff
@@ -883,124 +1076,361 @@
+                "34": [["DWN26", "A30+"], "_KISHA1", "_KISHA2", "_KISHA3", "_KISHA4", "_KISHA5", "_KISHA6", "DWN29"],
+                "10X": ["DW931", "LEMTA"],
+                "28X": ["DUB", "DWN29"],
+                "16X": ["OP", "BATED", "ERUDA"],
+                "34X": ["DUB", "DWN29"]
             },
-            "body": ["ROTEV"],
-            "draw": [["LEMTA", "ROTEV"], ["OE", "DWN29"], ["ERUDA", "ROTEV"], ["DWN29", "ROTEV"]]
+            "exitPoints": {
+                "ROTEV": ["ROTEV"]
+            },
+            "draw": [["LEMTA", "ROTEV"], ["_LIFFY21", "DWN29"], ["ERUDA", "ROTEV"], ["DWN29", "ROTEV"]]
         },
         "NEVRI": {
             "icao": "NEVRI",
             "name": "Nevri",
-            "suffix": {"10":"6E", "28":"1A", "16":"5M", "34":"1G"},
+            "suffix": {"10":"6E", "28":"1A", "16":"5M", "34":"1G", "10X":"6F", "28X":"1B", "16X":"5P", "34X":"1H"},
             "rwy": {
                 "10": [["DW551", "A30+"], "LEMTA"],
-                "28": [["OE", "A30+"], "DWN29"],
-                "16": [["DWS77", "A30+"], "BATED", "ERUDA"],
-                "34": [["DWN26", "A30+"], "DWN29"]
+                "28": [["OE", "A30+"], "_LIFFY1", "_LIFFY2", "_LIFFY3", "_LIFFY4", "_LIFFY5", "_LIFFY6", "_LIFFY7", "_LIFFY8", "_LIFFY9", "_LIFFY10", "_LIFFY11", "_LIFFY12", "_LIFFY13", "_LIFFY14", "_LIFFY15", "_LIFFY16", "_LIFFY17", "_LIFFY18", "_LIFFY19", "_LIFFY20", "_LIFFY21", "DWN29"],
+                "16": [["DWS77", "A30+"], "_BATED1", "_BATED2", "_BATED3", "_BATED4", "_BATED5", "_BATED6", "_BATED7", "_BATED8", "_BATED9", "_BATED10", "_BATED11", "_BATED12", "_BATED13", "_BATED14", "_BATED15", "_BATED16", "_BATED17", "_BATED18", "_BATED19", "_BATED20", "_BATED21", "_BATED22", "_BATED23", "_BATED24", "_BATED25", "_BATED26", "_BATED27", "_BATED28", "_BATED29", "BATED", "ERUDA"],
+                "34": [["DWN26", "A30+"], "_KISHA1", "_KISHA2", "_KISHA3", "_KISHA4", "_KISHA5", "_KISHA6", "DWN29"],
+                "10X": ["DW931", "LEMTA"],
+                "28X": ["DUB", "DWN29"],
+                "16X": ["OP", "BATED", "ERUDA"],
+                "34X": ["DUB", "DWN29"]
             },
             "body": ["NEVRI"],
-            "draw": [["DW551", "LEMTA", "NEVRI"], ["OE", "DWN29"], ["DWS77", "BATED", "ERUDA", "NEVRI"], ["DWN26", "DWN29", "NEVRI"]]
+            "exitPoints": {
+                "NEVRI": ["NEVRI"]
+            },
+            "draw": [["DW551", "LEMTA", "NEVRI"], ["DWS77", "_BATED1", "_BATED2", "_BATED3", "_BATED4", "_BATED5", "_BATED6", "_BATED7", "_BATED8", "_BATED9", "_BATED10", "_BATED11", "_BATED12", "_BATED13", "_BATED14", "_BATED15", "_BATED16", "_BATED17", "_BATED18", "_BATED19", "_BATED20", "_BATED21", "_BATED22", "_BATED23", "_BATED24", "_BATED25", "_BATED26", "_BATED27", "_BATED28", "_BATED29", "BATED", "ERUDA", "NEVRI"], ["_KISHA6", "DWN29", "NEVRI"]]
+        }
+    },
+    "stars": {
+        "OLAPO": {
```

Comment:
Right, what I'm trying to say is this:

When only using Runway 28... The `BAMLI departure` in use is `BAMLI 1B`. The `BAMLI arrival` in use is `BAML 1L` (note the lack of the `I` at the end).

Based on that, we should probably have:
```
"sids": {
    "BAMLI": {
        "suffixes": {"28": "1B"}
    }
},
"stars": {
    "BAML": {
        "suffixes": {"28": "1L"}
    }
}
```

This would result in the sim naturally using the same procedure identifiers as those listed in the IAIP, as well as allow us to use _both the BAMLI STAR and BAMLI SID_, which would not be possible if they both used `BAMLI` as their name. Win-win, unless there's a reason not to do it this way.